### PR TITLE
[DoctrineBridge] fixed default parameter value in UniqueEntityValidator

### PR DESCRIPTION
**Pull request by HeahDude**
| Q | A |
| --- | --- |
| Branch? | 3.1 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #19209 |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

**Comment by HeahDude** [comment]
Confirmed by @nicolas-grekas that the remaining failure in `Validator`'s suite for `appveyor` is unrelated.

**Comment by HeahDude** [comment]
@nicolas-grekas Is that normal that tests pass in high depth in travis? Shouldn't the validator dependency in the doctrine bridge be updated?

**Comment by HeahDude** [comment]
Ok I got it, tests are not running on 3.1, so maybe we should add a conflict?

File: src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php [hunk 1]
```diff
@@ -127,6 +127,10 @@ public function validate($entity, Constraint $constraint)
         $errorPath = null !== $constraint->errorPath ? $constraint->errorPath : $fields[0];
         $invalidValue = isset($criteria[$errorPath]) ? $criteria[$errorPath] : $criteria[$fields[0]];
 
+        if (is_object($invalidValue) && !method_exists($invalidValue, '__toString')) {
+            $invalidValue = sprintf('Object of class "%s" identified by "%s"', get_class($entity), implode(', ', $class->getIdentifierValues($entity)));
```
Comment (HeahDude) [inline src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php]: Why not use this in the default message by including the placeholder?

**Comment by fabpot** [comment]
Thank you @HeahDude.

**Comment by HeahDude** [comment]
Tests may fail after https://github.com/symfony/symfony/pull/19240, do they?

**Pull request merged**